.github/CI_MISC/build_compatibility_db.py

```diff
@@ -0,0 +1,112 @@
+
+import datetime
+import requests
+import sqlite3
+import re
+import os.path
+import os
+
+DB_NAME = 'states.db'
+
+def getStateLabels():
+	gh_token = os.environ['GITHUB_TOKEN']
+	r = requests.get("https://api.github.com/repos/jpd002/Play-compatibility/labels", headers={'Authorization': f'token {gh_token}'})
+
+	assert(r.status_code == 200)
+	assert(r.headers['content-type'] == "application/json; charset=utf-8")
+
+	res = []
+	for entry in r.json():
+		d = {}
+		d["name"] = entry["name"]
+		d["color"] = entry["color"]
+		res.append(d)
+	return res
+
+def requestGamesStateUpdates(url, res):
+	gh_token = os.environ['GITHUB_TOKEN']
+	r = requests.get(url, headers={'Authorization': f'token {gh_token}'})
+
+	assert(r.status_code == 200)
+	assert(r.headers['content-type'] == "application/json; charset=utf-8")
+
+	if(not r.json()):
+		return False
+
+	pattern = re.compile("\w{4}-\d{5}")
+	for entry in r.json():
+		d = {}
+		d["discId"] = entry["title"].split(" ")[0][1:-1]
+		d["labels"] = [state["name"] for state in entry["labels"]]
+		if(pattern.search(d["discId"])):
+			res.append(d)
+		else:
+			print(f"invalid entry: {entry['title']}")
+	return True
+
+def getLatestGamesStateUpdates():
+	URL = "https://api.github.com/repos/jpd002/Play-compatibility/issues?state=open&per_page=100&since={}&page={}"
+
+	today = datetime.date.today()
+	yesterday = today - datetime.timedelta(days=2)
+
+	i = 1
+	res = []
+	hasNext = True
+	while(hasNext):
+		print(f"Retrieving Page {i}")
```

Comment:
I would remove this as it doesn't really give interesting information (like for disc ids).